packages/next/package.json

```diff
@@ -79,7 +79,7 @@
     "chokidar": "3.5.1",
     "constants-browserify": "1.0.0",
     "crypto-browserify": "3.12.0",
-    "cssnano-simple": "2.1.0",
+    "cssnano-simple": "3.0.0",
```

Comment:
Do we need to bump our `postcss` version as well to `v8.2.15`? https://github.com/Timer/cssnano-simple/commit/507963a48072ee684174e5834fe94d4d379efdd0

We're currently using `v8.2.13` of `postcss`

Reply:
Nice catch! Updated 👍